tests/framework/extensions/workloads/cronjobs/create.go

```diff
@@ -0,0 +1,90 @@
+package cronjobs
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/pkg/api/scheme"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/unstructured"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/api/batch/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+var CronJobGroupVersionResource = schema.GroupVersionResource{
+	Group:    "batch",
+	Version:  "v1beta1",
```

Comment:
batch is on v1 not beta.

Reply:
Ah ok, but for some reason when I run it on v1. I get this error: `the server could not find the requested resource`

Reply:
Yeah @paynejacob I can only get the call to work using v1beta1. This what the view link looks like for my setup:
`https://israelhatest.qa.rancher.space/apis/batch/v1beta1/namespaces/default/cronjobs`